R/coef.R

```diff
@@ -28,6 +28,7 @@ coef.SLOPE <- function(object,
                        exact = FALSE,
                        simplify = TRUE,
                        sigma,
+                       only_nonzeros = FALSE,
```

Comment:
This functionality is already in another pull request. Please remove it from this pull request (sorry for not dealing with the previous pull request).